cleanlab/datalab/issue_manager/label.py

```diff
@@ -118,9 +126,38 @@ def _reset(self) -> None:
 
     def find_issues(
         self,
-        pred_probs: np.ndarray,
+        pred_probs: Optional[npt.NDArray] = None,
+        features: Optional[npt.NDArray] = None,
         **kwargs,
     ) -> None:
+        """Find label issues in the datalab.
+
+        Parameters
+        ----------
+        pred_probs :
+            The predicted probabilities for each example.
+
+        features :
+            The features for each example.
+        """
+        if pred_probs is None:
+            if features is None:
+                raise ValueError(
+                    "Either pred_probs or features must be provided to find label issues."
+                )
+            # produce out-of-sample pred_probs from features
```

Comment:
```suggestion
            # computing pred_probs from the given features
```

We are not getting out-of-sample pred_probs anymore

Reply:
@huiwengoh They are still out-of-sample pred_probs, just computed via leave-one-out cross-validation in a much more efficient matter.

@huiwengoh  Please check the math carefully, I wrote down the formula very hurriedly. Ensure the math/code is equivalent to doing leave-one-out cross-validation with a KNN model (you can even run a mini code test on toy dataset)

Reply:
This is the script I used to check the results. It gives about 5% differences between the class predicted by leave-one-out and by the KNN method. 
https://github.com/Steven-Yiran/cleanlab/blob/test/check_equal.ipynb

Reply:
Sorry there was a bug in my math if the current code does not agree with cross_val_predict. More strongly than the predictions agreeing is the pred_probs output by both methods should agree too.

However I spot many issues in your script so I'm not sure it is conclusive:
1. you are setting `k=5` and `n_neighbors=10` but these have to agree (or perhaps `k` should actually be `n_neighbors` + or - 1, need to check math closely to determine).
2. you set `cv=5` in the cross_val_predict, but it should be equal to the amount of data (or minus 1) to mimic leave-one-out cross-validation.

@huiwengoh please test this too and confirm, thanks!

Reply:
Thanks for pointing out my errors. I have updated the script above just for reference.

Reply:
@jwmueller your math is right, by omitting the datapoint itself it is equivalent to doing a KNN with `k-1` neighbours. 

Perhaps the something we want to change is to add one to the `n_neighbors` arg to the KNN initializer (ie. `knn = KNeighborsClassifier(n_neighbors=self.k + 1)`) so we adjust for the correct number of neighbours after omitting one datapoint later.